### DISCUSSION TITLE
My page works great locally but messes up when opened using GitHub Pages

### DISCUSSION THREAD

**Discussion by weary123hi** [community/community#22247] [Other Feature Feedback, Questions, & Ideas]
<p>Hey all, as you can tell I’m rather new to all of this. Upon finishing the first half of my website, everything seemed to work great. I could open it and interact with it on my local PC without any issues. The problem, however, is that the second I open it up using GitHub Pages everything goes kaput. The first page and “contact” page work great, but my “merch” and “activities” page are all messed up and are missing their appropriate background images and have unusual formatting. Any help would be much appreciated!</p>
<p>My repo: <a href="https://github.com/weary123hi/va821_website" rel="noopener nofollow ugc">https://github.com/weary123hi/va821_website</a></p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/weary123hi">@weary123hi</a>, welcome to the GitHub Support Community!</p>
<p>GitHub Pages is a case sensitive file host. It looks like you’re referencing your image files like this:</p>
<pre><code class="lang-auto">&lt;img src="images/Activities1.jpg"&gt;
</code></pre>
<p>However, that file is actually named <code>Activities1.JPG</code>. You’ll need to edit your HTML so it matches the case in your filenames for your images to load correctly.</p>

**Comment by weary123hi** [comment]
<p>Oh, I see. Thank you!</p>
